Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs

```diff
@@ -979,11 +979,19 @@ public static void CreateUpdateRunSetFromJSON(RunsetExecutor runsetExecutor, Gin
                 }
             }
 
-            if (gingerExecConfig.Runset.RerunConfigurations != null && gingerExecConfig.Runset.RerunConfigurations.Active)
+            if (gingerExecConfig.Runset.RerunConfigurations != null)
             {
-                runSetConfig.ReRunConfigurations.Active = gingerExecConfig.Runset.RerunConfigurations.Active;
-                runSetConfig.ReRunConfigurations.RerunLevel = (global::Ginger.Run.eReRunLevel)gingerExecConfig.Runset.RerunConfigurations.RerunLevel;
-                runSetConfig.ReRunConfigurations.ReferenceExecutionID = gingerExecConfig.Runset.RerunConfigurations.ReferenceExecutionID;
+                if(gingerExecConfig.Runset.RerunConfigurations.Active)
+                {
+                    runSetConfig.ReRunConfigurations.Active = gingerExecConfig.Runset.RerunConfigurations.Active;
```

Comment:
Move this line out if as same line exist in else part

Reply:
done